### PR TITLE
docs: update the rust version from version.yaml

### PR DESCRIPTION
**Pull request by houstar**
Fixes: #6539

### CODE REVIEW

File: docs/install/kata-containers-3.0-rust-runtime-installation-guide.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Follow the [`kata-deploy`](../../tools/packaging/kata-deploy/README.md).
 
 * Download `Rustup` and install  `Rust`
     > **Notes:**
-    > Rust version 1.62.0 is needed
+    > For Rust version, please see version.yaml file's rust section.
```
Comment (Tim-0731-Hzt) [inline docs/install/kata-containers-3.0-rust-runtime-installation-guide.md:52]: I think you could add a link for the `version.yaml`

**Review by GabyCT — APPROVED** [review]

File: docs/install/kata-containers-3.0-rust-runtime-installation-guide.md (hunk 1 above)
Comment (GabyCT) [reply docs/install/kata-containers-3.0-rust-runtime-installation-guide.md:52]: @houstar just please fix the typo and the rest is lgtm

File: docs/install/kata-containers-3.0-rust-runtime-installation-guide.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Follow the [`kata-deploy`](../../tools/packaging/kata-deploy/README.md).
 
 * Download `Rustup` and install  `Rust`
     > **Notes:**
-    > Rust version 1.62.0 is needed
+    > For Rust version, please see versions.yaml file's rust section.
```
Comment (Tim-0731-Hzt) [inline docs/install/kata-containers-3.0-rust-runtime-installation-guide.md:52]: What I mean is you can do like this
please see [`versions.yaml`](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml) file's rust section.

Comment (houstar) [reply docs/install/kata-containers-3.0-rust-runtime-installation-guide.md]: done, tks

**Review by Tim-0731-Hzt — APPROVED** [review]
Thanks @houstar LGTM

**Pull request merged**